catalyst/core/callbacks/early_stop.py

```diff
@@ -0,0 +1,54 @@
+from catalyst.core import _State, Callback, CallbackOrder
+
+
+class CheckRunCallback(Callback):
+    def __init__(self, num_loader_steps: int = 1, num_epoch_steps: int = 1):
```

Comment:
can you return defaults to 3? it was super-secure

Reply:
fixed with env args